tests/playwright/account.spec.js

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from "@playwright/test";
+import { AccountPage } from "./pages/AccountPage";
+import { RegisterPage } from "./pages/RegisterPage";
+import { BasePage } from "./pages/BasePage";
+
+test("Register account", async ({ page }) => {
+  const registerPage = new RegisterPage(page);
+
+  const email = `wayne+${crypto.randomUUID()}@enterprises.com`;
+  const password = "IronManSucks.91939";
+
+  await registerPage.register("Bruce", "Wayne", email, password);
+
+  await expect(page.getByTestId("account-menu")).toContainText("Bruce");
+  await new BasePage(page).screenshot('account-registered');
+});
+
+test("Logout account", async ({ page }) => {
```

Comment:
Maybe use the logout from here? https://github.com/rapidez/core/blob/45c387e38196c4bc9ac560bc85ec39b3e95fab65/tests/playwright/checkout.spec.js#L34